chapters/feed-forward.tex

```diff
@@ -14,20 +14,45 @@ \chapter{Feed-Forward Layers}
 \AY{Other activation functions. *LU}
 \AY{Unveiling chain of thought GELU---multiplication!!!}
 
-\begin{definition}{$\ReLU$}{}
-    The activation function $\ReLU : \R^d\to \R^d$, which stands for \say{Rectified Linear Unit}, is a standard component of feed-forward layers, where
+\section{Activation functions}
+
+The activation function component of feed-forward layers enables them to approximate non-linear functions. We define two popular activation functions. Other notable activation functions include sigmoid, softmax, hyperbolic tangent, Exponential Linear Unit (ELU) and Gated Linear Unit (GLU).
+
+\begin{definition}{ReLU}{}
+    Rectified Linear Unit (ReLU) $: \R^d\to \R^d$, is a non-linear activation function as defined below \cite{Fukushima1975}.
```

Comment:
$\ReLU : \R^d \to R^d$ to match next definition

Reply:
Thanks—will do.